narwhal/source_code_sharing/update_liferay_dxp_source_code_of_hotfixes.sh

```diff
@@ -305,11 +330,15 @@ function print_help {
 }
 
 function process_argument_ignore_zip_files {
+	IGNORE_ZIP_FILES_PRESISTENT_FILE="$(dirname "$(readlink /proc/$$/fd/255 2>/dev/null)")/ignore_zip_files_persistent.txt"
+	IGNORE_ZIP_FILES_EXPANDING_FILE="${LIFERAY_COMMON_DOWNLOAD_CACHE_DIR}/ignore_zip_files_expanding.txt"
```

Comment:
IGNORE_ZIP_FILES_CACHE_FILE would be better name.

Reply:
And it should be contained relatively, so if there is a test and a production job on the same jenkins node, they do not conflict.